### PR TITLE
Fix updateReinfState to no longer have cross reinforcements

### PR DESCRIPTION
**Pull request by wurzel0701**
## What type of PR is this.
1. [X] Bug
2. [ ] Enhancement

### What have you changed and why?
Information: Fixed updateReinfState to avoid CSAT reinforcing NATO and the other way around

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the Mission in Singleplayer?
2. [ ] Have you loaded the Mission in a Dedicated Server?

### Is further testing or are further changes required?
1. [ ] No
2. [X] Yes (Please provide further detail below.)

********************************************************
Notes: Untested as this is a quickfix

### CODE REVIEW

File: A3-Antistasi/functions/Base/fn_markerChange.sqf [hunk 1]
```diff
@@ -92,7 +92,7 @@ else
 	//End ========================================================================
 };
 
-[_markerX] call A3A_fnc_updateReinfState;
+[_markerX, _winner, _looser] call A3A_fnc_updateReinfState;
```
Comment (jaj22) [inline A3-Antistasi/functions/Base/fn_markerChange.sqf]: "looser" is a misspelling of "loser", in context. It's common but it still triggers me.

**Comment by jaj22** [comment]
General issue with using one function for two separate operations, namely handling marker changes and timed updates. I'd recommend at least making that duality clear in the params: Have a second param that's either [winner, loser] or null.

File: A3-Antistasi/functions/Base/fn_markerChange.sqf (hunk 1 above)
Comment (wurzel0701) [reply A3-Antistasi/functions/Base/fn_markerChange.sqf]: Yeah, I thought I go with it, but I should probably fix it

File: A3-Antistasi/functions/Garrison/fn_updateReinfState.sqf [hunk 2]
```diff
@@ -1,78 +1,97 @@
...
-_isAirport = _marker in airportsX;
+    if(_ratio != 1) then
+    {
+        //In need of reinforcements
+        if(!_isAirport) then
+        {
+            //Airports don't get send reinforcements
+            if(_index == -1) then
+            {
+                //Add new
```
Comment (Spoffy) [inline A3-Antistasi/functions/Garrison/fn_updateReinfState.sqf]: Fine with these, as they describe the line below well enough :D

File: A3-Antistasi/functions/Garrison/fn_updateReinfState.sqf [hunk 3]
```diff
@@ -1,78 +1,97 @@
...
 
-_reinfMarker = if(_side == Occupants) then {reinforceMarkerOccupants} else {reinforceMarkerInvader};
-_canReinf = if(_side == Occupants) then {canReinforceOccupants} else {canReinforceInvader};
+    private _isAirport = _marker in airportsX;
+    private _index = _reinfMarker findIf {(_x select 1) == _marker};
 
-_isAirport = _marker in airportsX;
+    if(_ratio != 1) then
+    {
+        //In need of reinforcements
```
[the remark this answers is not in the document]
Comment (wurzel0701) [reply A3-Antistasi/functions/Garrison/fn_updateReinfState.sqf]: Fixed the comments

File: A3-Antistasi/functions/Garrison/fn_updateReinfState.sqf (hunk 2 above)
Comment (wurzel0701) [reply A3-Antistasi/functions/Garrison/fn_updateReinfState.sqf]: Improved the comment a bit

File: A3-Antistasi/functions/Garrison/fn_updateReinfState.sqf [hunk 4]
```diff
@@ -1,78 +1,97 @@
...
+        if(!_isAirport) then
+        {
+            //Airports don't get send reinforcements
```
[the remark this answers is not in the document]
Comment (wurzel0701) [reply A3-Antistasi/functions/Garrison/fn_updateReinfState.sqf]: Fixed

**Pull request merged**